packages/offix-datasync-codegen/declarations.d.ts

```diff
@@ -0,0 +1 @@
+declare module 'graphql-metadata';
```

Comment:
Is there issue with typings for metadata?

Reply:
Yes

Reply:
Cc @craicoverflow Let's fix them then